src/Ilios/CoreBundle/Entity/Offering.php

```diff
@@ -222,6 +222,9 @@ class Offering implements OfferingInterface
     public function __construct()
     {
         $this->deleted = false;
+        $this->startDate = getdate();
+        $this->endDate = getdate();
+        $this->lastUpdatedOn = getdate();
```

Comment:
startDate and endDate should not have constructor values.  They will need to be set by the user or else throw an error.

getdate() is not the correct initializer for these fields.  it should be ```php $this->createdAt = new \DateTime();```